src/actions/index.js

```diff
@@ -12,26 +12,28 @@ export function fetchSearchIndexSuccess(searchIndex) {
     type: actionTypes.FETCH_SEARCH_INDEX_SUCCESS,
     searchIndex,
   };
-}
+} 
 
 export function fetchSearchIndexFailure() {
   return {
     type: actionTypes.FETCH_SEARCH_INDEX_FAILURE,
   };
 }
 
-export const getSearchResultsRequest = searchInput => ({
-  type: actionTypes.GET_SEARCH_RESULTS_REQUEST,
-  searchInput,
-});
+export function fetchSubjectsRequest(){
```

Comment:
Put a space between the parens and the opening curly brace. Linting should take care of this for you. I recommend installing a plugin for your IDE that highlights linting errors as you code (if you use VS Code I can point you to some good ones).

Reply:
This action creator is also going to need an input - specifically the input that the `fetchSubjects` effects requires (`schoolId`). The user will click a school, which will dispatch this action with the `schoolId` of the school that the user clicked.

Reply:
I'll definitely get a linter. I'll add the schoolId input